hack/verify-spelling.sh

```diff
@@ -20,7 +20,7 @@ set -o pipefail
 DIR=$(dirname $0) 
 
 # Install tools we need
-go install ${DIR}/../../../github.com/client9/misspell/cmd/misspell
+# go install ${DIR}/../../../github.com/client9/misspell/cmd/misspell
```

Comment:
This is already installed by the script above. I commented it out to limit the diff to upstream.